R/verb-expand.R

```diff
@@ -0,0 +1,44 @@
+#' @importFrom tidyr expand
+#' @export
+expand.tbl_lazy <- function(data, ..., .name_repair = "check_unique") {
+  # TODO sort output?
+  # TODO wait for bugfix: distinct() ignores groups
+  # https://github.com/tidyverse/dbplyr/issues/535
+  dots <- quos_auto_name(quos(...))
+  dots <- purrr::discard(dots, rlang::quo_is_null)
+  names(dots) <- vctrs::vec_as_names(names(dots), repair = .name_repair)
+
+  if (is_empty(dots)) {
+    # TODO what should be returned here?
+    abort("No columns selected")
```

Comment:
I think it's fine for this to be an error; just needs to be a bit more explicit (and follow https://style.tidyverse.org/error-messages.html). Maybe something like "Must supply variables in `...`" ?